### PR TITLE
Add Travis CI setting

### PR DESCRIPTION
**Pull request by SarahhhC**
@ethkim @nlpenguin @ifndefdeadmau5 @djflexible 

Travis CI setting Added. Please review the code.

### CODE REVIEW

File: src/server/controllers/users.controller.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ let mockData = {
 };
 
 export function saveTest(req, res, next) {
-  let user = new user(mockData);
-  user.save((err) => {
+  let userSample = new user(mockData);
```
Comment (ethkim) [inline src/server/controllers/users.controller.js]: I'm a bit confused. The contents in this file seem to be primarily for testing. Shouldn't it be in src/test directory then?

File: src/test/node/test.js [hunk 2]
```diff
@@ -1,17 +1,14 @@
-'use strict';
-
 import test from 'tape';
 
-test('A passing test', (assert) => {
-  assert.pass('This test will pass.');
-  assert.end();
+test('A passing test', (t) => {
```
Comment (ethkim) [inline src/test/node/test.js]: Could you leave comments above each of these tests that these are just sample tests?

**Comment by ethkim** [comment]
Thanks Sarah!
At the top of each test files, could you comment on what types of tests are to be written in each corresponding file? It's unclear where I should write it, if I were to add a test.

File: src/test/node/test.js (hunk 2 above)
Comment (SarahhhC) [inline src/test/node/test.js]: Yes I added some comments!

File: src/server/controllers/users.controller.js (hunk 1 above)
Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: I think the method name was confused, it's for inserting mock data.
And user.js is the test for this method.

**Comment by ethkim** [comment]
LGTM

Thanks Sarah!

**Comment by djflexible** [comment]
LGTM

**Pull request merged**